starter-cli/src/main/java/io/micronaut/starter/cli/command/CreateLambdaBuilderCommand.java

```diff
@@ -0,0 +1,284 @@
+        out("Choose the target JDK. (enter for default)");
+        return getEnumOption(
+                versions,
+                jdkVersion -> Integer.toString(jdkVersion.majorVersion()),
+                versions.length > 0 ? versions[0] : JdkVersion.DEFAULT_OPTION,
+                reader
+        );
+    }
+
+    JdkVersion[] jdkVersionsForDeployment(LambdaDeployment deployment) {
+        switch (deployment) {
+            case NATIVE_EXECUTABLE:
+                return new JdkVersion[] {
+                        JdkVersion.JDK_17,
+                        JdkVersion.JDK_11,
+                };
+            default:
+            case FAT_JAR:
+                return new JdkVersion[] {
+                        JdkVersion.JDK_11,
+                        JdkVersion.JDK_8,
+                };
+        }
+    }
+
+    protected ApplicationType applicationTypeForCodingStyle(CodingStyle codingStyle) {
+        switch (codingStyle) {
+            case HANDLER:
+                return ApplicationType.FUNCTION;
+            default:
+            case CONTROLLERS:
+                return ApplicationType.DEFAULT;
+
+        }
+    }
+
+    protected Language[] languagesForDeployment(LambdaDeployment deployment) {
+        return deployment == LambdaDeployment.NATIVE_EXECUTABLE ?
+                Stream.of(Language.values())
+                        .filter(graalVMFeatureValidator::supports)
+                        .toArray(Language[]::new) :
+                Language.values();
+    }
+
+    protected Feature getApiTrigger(LineReader reader) {
+        Feature defaultFeature = features.stream().filter(f -> f instanceof AmazonApiGateway).findFirst()
+                .orElseThrow(() -> new ConfigurationException("default feature " + LambdaFunctionUrl.NAME + " not found"));
+        out("Choose your trigger. (enter for " + defaultFeature.getTitle() + ")");
+        return getFeatureOption(
+                apiTriggerFeatures(features),
+                Feature::getTitle,
+                defaultFeature,
+                reader);
+    }
+
+    protected Feature getTrigger(LineReader reader) {
+        Feature defaultFeature = features.stream().filter(f -> f instanceof LambdaFunctionUrl).findFirst()
+                .orElseThrow(() -> new ConfigurationException("default feature " + LambdaFunctionUrl.NAME + " not found"));
+        out("Choose your trigger. (enter for " + defaultFeature.getTitle() + ")");
+        return getFeatureOption(
+                triggerFeatures(features),
+                Feature::getTitle,
+                defaultFeature,
+                reader);
+    }
+
+    protected Architecture getArchitecture(LineReader reader) {
+        out("Choose your Lambda Architecture. (enter for Java ARM)");
+        return getEnumOption(
+                Architecture.class,
+                a -> a.toString() +  (a == Architecture.ARM ? " Better performance and 20% cheaper" : ""),
```

Comment:
I'd remove the actual figure, since this can change in the future.
```suggestion
                a -> a.toString() +  (a == Architecture.ARM ? " Better performance and lower-priced" : ""),
```